model/src/main/kotlin/AnalyzerResult.kt

```diff
@@ -86,8 +90,21 @@ class AnalyzerResultBuilder(
     }
 
     fun addResult(projectAnalyzerResult: ProjectAnalyzerResult) = this.apply {
-        projects.add(projectAnalyzerResult.project)
-        packages.addAll(projectAnalyzerResult.packages)
-        errors[projectAnalyzerResult.project.id] = projectAnalyzerResult.errors
+        val duplicateProject = projects.find { it.id == projectAnalyzerResult.project.id }
+        if (duplicateProject != null) {
+            val error = "Found multiple projects with the same id '${duplicateProject.id}', not adding the " +
```

Comment:
May better: "Multiple projects with the same id ... found. Not adding the project defined in '${duplicateProject.definitionFilePath}' to the analyzer results as it duplicates the project defined in '${projectAnalyzerResult.project.definitionFilePath}'."

That way it's clear from the error message which is the duplicate definition file and which is the original.

Reply:
Done.